corehq/motech/openmrs/tasks.py

```diff
@@ -336,7 +336,7 @@ def import_patients():
 
 @task(queue='background_queue')
 def poll_openmrs_atom_feeds(domain_name):
-    for repeater in OpenmrsRepeater.by_domain(domain_name):
+    for repeater in SQLOpenmrsRepeater.objects.by_domain(domain_name):
```

Comment:
@millerdev @kaapstorm What would be a good testing strategy for this change?

I have tested its internal functions that pass repeater on a django shell that they return same output for both the models. Was curious what a formal test for it should look like?

Reply:
Is this similar to 1123d58849ee1673a5e6df2db57166426f50c16a? (That's a commit on a branch that has not been PR'd yet, and will be rebased before then, FYI.) In that commit I added a unit test for `LookupTableManager.by_domain`, which is minimal but does exercise the method.

https://github.com/dimagi/commcare-hq/blob/1123d58849ee1673a5e6df2db57166426f50c16a/corehq/apps/fixtures/tests/test_models.py#L27-L31

Is that what you're asking, or are you asking more specifically how to test `poll_openmrs_atom_feeds` after this change?

Reply:
`SQLRepeaters.objects.by_domain` was automatically tested with the tests which were added for the changes.
I was more specifically concerned about testing `poll_openmrs_atom_feeds`

Reply:
You could write a happy-path test for that task function since that would exercise the query you're changing here. Be sure to verify that the test works before and after this query change has been applied.

Reply:
Tested with both couch and SQL repeaters https://github.com/dimagi/commcare-hq/pull/32021/commits/616eb1b9d0e34af1271d3e1a31aecff0cda216a5